Add Doppler integration (ECOINT-6) (#2460)
DataDog/integrations-extras: doppler/README.md

# Agent Check: Doppler

## Overview

> [Doppler][1] is a secrets manager designed with both security and developer productivity in mind.

Author: Thanks! I've updated this section with more of those specifics.

This integration allows Activity Logs from [Doppler][1] to be streamed to Datadog, allowing you to monitor changes to your workplace.

## Setup

> 

Reviewer: Can you please add clear directions a user would take to install the integration? For example, go to the Doppler tile on the integrations page, click connect accounts, you will be redirected to doppler to log in, .. 

This should walk through the full oauth flow a user will take. I recommend testing the oauth flow and creating the steps on the steps you take!

Author: Absolutely, I re-tested the full oauth flow and provided more detail here.

1. Click **Connect Accounts** on the Doppler integration tile to connect Datadog with Doppler.
2. Log into Doppler or create an account if you have not already.
3. Select the Doppler workplace that you would like to configure. This step will automatically be skipped if you only have one workplace.

Once the installation is complete, Doppler Activity Logs will automatically start flowing to Datadog.

### Configuration

> The integration will automatically send all Doppler Activity Logs to Datadog, no further configuration is available at this time.

Reviewer 2: ```suggestion
The integration will automatically send all Doppler Activity Logs to Datadog. No further configuration is available at this time.
```

### Validation

> During installation, Doppler creates a test log to verify that the installed credentials work properly. Verify that this test log is present in your logs to verify the installation.

Reviewer 2: ```suggestion
During installation, Doppler creates a test log to verify that the installed credentials work properly. Confirm that this test log is present in your logs to verify the installation.
```

### Uninstallation

> - Go to your [Doppler workplace settings][4] and disconnect the Datadog integration

Reviewer 2: ```suggestion
- Go to your [Doppler workplace settings][4] and disconnect the Datadog integration.
```

# Agent Check: Doppler

## Overview

[Doppler][1] is a secrets manager designed with both security and developer productivity in mind.
Doppler allows you to securely store secrets (for example: API keys, database credentials, and other sensitive values) and deliver them to your applications.

> This integration allows Activity Logs from [Doppler][1] to be streamed to Datadog, allowing you to monitor changes to your workplace.

Reviewer: ```suggestion
This integration sends Activity Logs from [Doppler][1] to Datadog, allowing you to monitor changes to your workplace.
```

Can you provide examples on changes?

Author: Absolutely 👍
